masq/src/communications/broadcast_handler.rs

```diff
@@ -16,6 +19,21 @@ pub trait BroadcastHandle: Send {
     fn send(&self, message_body: MessageBody);
 }
 
+pub struct BroadcastHandleInactive {}
+
+impl BroadcastHandle for BroadcastHandleInactive {
+    fn send(&self, message_body: MessageBody) {
+        drop(message_body); //simply dropped (unless we find a better use for such a message)
```

Comment:
You could remove this line (except for the comment, that's good) and put an underscore at the beginning of the variable name.